lib/views/LineGraph.js

```diff
@@ -25,33 +26,47 @@ class LineGraph extends Graph {
             .y(d => this.yRange(d.y_value))
             .interpolate(this.interpolate);
 
-        var line = this.d3Svg.selectAll('path.line')
-                    .data([this.dataSeries.data]);
+        var data = Utils.splitOnMissingValues(this.dataSeries.data, ['x_value', 'y_value']);
```

Comment:
Yes. Longer term, I'd like to rip out the DataSeries models altogether. Most of the graphs require one specific type of data series (line/scatter now support 2). So I'd rather not split the behavior across two modules.